src/eventHandlers/FunctionEnvironmentReloadHandler.ts

```diff
@@ -63,3 +84,7 @@ export class FunctionEnvironmentReloadHandler extends EventHandler<
         return response;
     }
 }
+
+function isPathEqual(path1: string, path2: string): boolean {
+    return path.relative(path1, path2) === '';

```

Comment:
Paths in linux are case sensitive. Could there be a case where these 2 paths have different cases?

Reply:
That's a big part of why I use the `path.relative` method instead of comparing the string paths directly. The `path.relative` method handles all the weird edge cases (like case sensitivity) for us